Library/ViewModels/RewardsCollectionViewModel.swift

```diff
@@ -27,6 +31,17 @@ RewardsCollectionViewModelInputs, RewardsCollectionViewModelOutputs {
     )
       .map(first)
       .map { $0.rewards }
+
+    let selectedReward = reloadDataWithRewards
+      .takePairWhen(self.rewardSelectedIndexProperty.signal.skipNil())
+      .map { rewards, index in rewards[index] }
+
+    self.goToPledge = Signal.combineLatest(self.configureWithProjectProperty.signal.skipNil(),
+      selectedReward,
+      self.configureWithRefTagProperty.signal)
+      .map { project, reward, refTag in
+        return GoToPledgeData(project: project, reward: reward, refTag: refTag)
```

Comment:
A suggestion here: we usually use names starting with `goTo` to define an action that brings the user to a different screen in the app. Maybe `PledgeData(project:reward:refTag)` would be better?

Reply:
Yep that's better, thanks for the suggestion!